ChangeLog

```diff
@@ -17,6 +17,9 @@ Release date: TBA
 
   Closes pylint-dev/pylint#9811
```

Comment:
Not strictly related with your changes, but would you mind deleting the duplicate fragment just above yours? It snuck in after merging maintenance back into main.